com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -63,7 +63,7 @@ internal void Cleanup()
 
         const int kMaxPoolWidth = 1 << 11; // 2048 texels is a d3d11 limit for tex3d in all dimensions
 
-        int m_AllocationSize;
+        static int m_AllocationSize;
```

Comment:
It's a bit weird this is static given it is added in the constructor of the pool itself

Reply:
I agree and it's true that it's dangerous but I think that this actually uncovers an issue.
This allocation size is set at runtime init which means that you can theoretically change it after baking.
It's currently hard coded in HDRP init code. Would you mind if I made it a constant like we have for probe indices? Or is this something that we want to really expose to users later on?
In any case, the reason why it's static is because I wanted to precompute that at bake time because it feels like the right place to do that and from there I don't have access to runtime pool etc.

Reply:
I think is fine to move for now as constant, I don't see much need to have it as settable for user for now, can always change later.